vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb

```diff
@@ -32,14 +32,19 @@ def self.recent
 
     def mark_orphaned!
       update!(orphaned_at: Time.current)
+      attempt_order_complete!
+    end
+
+    def attempt_order_complete!
+      order_detail.complete! if order_completable?
     end
 
     def orphan?
       orphaned_at?
     end
 
     def complete?
```

Comment:
Where else does `complete?` get used? It feels a little odd for an orphaned occupancy to be considered `complete?`.